index.html

```diff
@@ -0,0 +1,267 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.2/css/all.min.css"
+        integrity="sha512-HK5fgLBL+xu6dm/Ii3z4xhlSUyZgTT9tuc/hSrtw6uzJOvgRr2a9jyxxT1ely+B+xFAmJKVSTbpM/CuL7qxO8w=="
+        crossorigin="anonymous" />
+
+    <link rel="stylesheet" href="./doc-css/docpage.css">
+    <title>Space ui</title>
+
+</head>
+
+<body>
+    <div class="pagelayout-grid">
+        <nav class="navbar">
+
+        </nav>
+
+        <div class="sidebar">
+            <ul class="sidebar-items">
+                <li class="sidebar-item"> <a href="">Avatar</a></li>
+                <li class="sidebar-item"> <a href="">Alert</a></li>
+                <li class="sidebar-item"> <a href="">Badge</a></li>
+                <li class="sidebar-item"> <a href="">Button</a></li>
+                <li class="sidebar-item"> <a href="">Card</a></li>
+                <li class="sidebar-item"> <a href="">Image</a></li>
+                <li class="sidebar-item"> <a href="">Input</a></li>
+
+            </ul>
+        </div>
+        <div class="main">
+            <section class="avatar-component">
+
+                <h1>
+                    Avatar
+                </h1>
+                <p>Use the Avatar Component as a graphical representation of a person through a profile picture or a string with initials</p>
+
+                <h2>
+                    Example of Image avatar
+                </h2>
+
+                <p>Avatar is available in 5 different sizes. You can use image in Avatar.</p>
+
+                <div class="component-display-container">
+                    <div class="avatar avatar-xl margin-right-1">
+                        <img src="./images/girls.jpg" class="image-responsive image-round " alt="">
+                    </div>
+
+                    <div class="avatar avatar-l margin-right-1">
```

Comment:
You can use class name ---> avatar-lg  instead of avatar-l  in place of just l for size. I made this change too.
sm for s , md for m , and x-sm for x-s .

Reply:
The rest is good to go.

Reply:
thank you!